hw_os/cat.py

```diff
@@ -0,0 +1,35 @@
+#!./venv/bin/python
```

Comment:
Данная строка "скажет" командной оболочке, что нужно взять интерпретатор по этому адресу. Проблема в том, что текущее активное окружение не обязательно будет называться `venv` и не обязательно будет находиться в текущей папке. Для универсальность используют вот такую строчку:
```suggestion
#!/usr/bin/env python
```